pom.xml

```diff
@@ -293,9 +204,6 @@
 							<goal>integration-test</goal>
 						</goals>
 						<configuration>
-							<includes>
```

Comment:
We create pull request against your own fork master for now.  After we have released liberty-maven-plugin 2.0, we will create another PR from your fork master to the real master.